### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by hemangsk**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
+    Copyright (C) <year>  <name of author>
```
Comment (meetmangukiya) [inline LICENSE.md]: Fill this namespace

**Comment by meetmangukiya** [comment]
Also no closes `statement` in the commit body.

**Comment by meetmangukiya** [comment]
#9

**Comment by hemangsk** [comment]
Apologies 😞 
 I think i did something wrong, I pushed changes with --force and PR got closed. Will create a new PR. Figuring out what happened

**Pull request closed without merging**